CONTRIBUTING.md

```diff
@@ -28,24 +27,14 @@ This repository contains some projects used to display, explain components built
 
 If you want to use these projects, you must:
 
-```sh
-cd examples/demo
-npm i
-```
-
 ```sh
 cd storybook/design-system
-npm i
+npm ci
 ```
 
 ```sh
 cd storybook/storybook
-npm i
-```
-
-```sh
-cd storybook/styles
```

Comment:
Why not remove line 24 as well?

Reply:
already done here : https://github.com/AxaGuilDEv/react-toolkit/pull/692